halo/evmstaking2/keeper/keeper.go

```diff
@@ -162,6 +165,9 @@ func (k Keeper) Deliver(ctx context.Context, _ common.Hash, elog evmenginetypes.
 		return errors.Wrap(err, "insert evm event")
 	}
 
+	log.Debug(ctx, "Scheduled a new staking event")
```

Comment:
best log this where you can add more context, event type, key fields, etc. This doesn't provide enough information, it only logs that something happened, not what actually happened, it will not help when debugging

Reply:
Not sure this is easily possible as we indeed have no additional information ad this point: the event is and stays unparsed until we start processing it, right?

Reply:
then lets remove the log as it doesn't add sufficient value